HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,571 @@
+                "Created": {},
+                "EntryType": {},
+                "EventGroupId": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Message": {
+                    "ReadRequirement": "Recommended"
+                },
+                "MessageId": {},
+                "MesageArgs": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Severity": {}
+            }
+        },
+        "LogService": {
+            "PropertyRequirements": {
+                "DateTime": {},
+                "DateTimeLocalOffset": {},
+                "Entries": {},
+                "LogEntryType": {},
+                "LogPurposes": {
+                    "ReadRequirement": "Recommended"
+                },
+                "ServiceEnabled": {
+                    "ReadRequirement": "Recommended"
+                }
+            },
+            "ActionRequirements": {
+                "ClearLog": {
+                    "ReadRequirement": "Recommended"
+                }
+            }
+        },
+        "Manager": {
+            "PropertyRequirements": {
+                "DateTime": {
+                    "ReadRequirement": "Mandatory",
+                    "WriteRequirement": "Recommended"
+                },
+                "DateTimeOffset": {
+                    "ReadRequirement": "Mandatory",
+                    "WriteRequirement": "Recommended"
+                },
+                "ManagerType": {},
+                "UUID": {},
+                "Status": {
+                    "PropertyRequirements": {
+                        "Health": {},
+                        "State": {}
+                    }
+                },
+                "FirmwareVersion": {},
+                "NetworkProtocol": {
+                    "ReadRequirement": "Supported"
+                },
+                "EthernetInterfaces": {},
+                "LogServices": {},
+                "Links": {
+                    "PropertyRequirements": {
+                        "ManagerForChassis": {
+                            "ReadRequirement": "Supported"
+                        }
+                    }
+                }
+            },
+            "ActionRequirements": {
+                "Reset": {
+                    "Parameters": {
+                        "ResetType": {
+                            "ParameterValues": [
+                                "ForceRestart"
+                            ]
+                        }
+                    }
+                }
+            }
+        },
+        "ManagerAccount": {
+            "PropertyRequirements": {
+                "AccountTypes": {},
+                "Enabled": {},
+                "Links": {
+                    "PropertyRequirements": {
+                        "Role": {}
+                    }
+                },
+                "Password": {},
+                "PasswordChangeRequired": {},
```

Comment:
Might want to consider PasswordChangeRequired and PasswordExpiration as "Recommended"; not sure how widely used it is at the moment.

Reply:
Agreed on both.  I was thinking the ChangeRequired was necessary - but no if it's absent then the change-required functionality isn't present, either.